README.md

```diff
@@ -1,12 +1,12 @@
-# Getting up to speed with Python
+# Introduction to Python
```

Comment:
This is the book title, could you undo the change here?

Reply:
Yes I will change this back.